.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: php
 
 php:
+    - 7.1
```

Comment:
missing composer in the pull-up :thinking:

Reply:
Nothing changed in the `composer.json` but I need to pull-up the `akeneo/api-php-client` too

Reply:
I will do a composer update once the CE version of the php-client is merged.
With probably some fixes for phpunit 8